bin/resources/shaders/opengl/convert.glsl

```diff
@@ -252,16 +256,22 @@ void ps_convert_rgba_8i()
     uvec2 subblock = pos & uvec2(7u, 1u);
     uvec2 coord = block | subblock;
 
+    // Compensate for potentially differing page pitch.
+	uvec2 block_xy = coord / uvec2(64u, 32u);
+	uint block_num = (block_xy.y * (DBW / 128u)) + block_xy.x;
+	uvec2 block_offset = uvec2((block_num % (SBW / 64u)) * 64u, (block_num / (SBW / 64u)) * 32u);
+	coord = (coord % uvec2(64u, 32u)) + block_offset;
```

Comment:
We should probably clang format the gl shaders, they use spaces instead of tabs.